### DISCUSSION TITLE
Please Help Me， I couldn't find where to resubmit my information.

### DISCUSSION THREAD

**Discussion by YvMiao1** [community/community#123501] [GitHub Education]
### Hi everyone,

My last account was banned, and I was given this information after appealing, but I couldn't find where to resubmit my information.
![屏幕截图(312)](https://github.com/community/community/assets/169591261/20ec57d7-2912-4685-b67c-150c91567f6c)

**Comment by ghost** [comment]
## Solution 🌟
- To Re-submit your information repeat the process of filing your application.
- You can simply visit the site of - https://education.github.com/benefits
- There it will be asking to re-verify your application.
- Enter you all details and get verified ✅ again.
> Good Luck 😊👍

**Comment by IP80808080** [comment]
Resubmitting Information for GitHub Education:

While there isn't a dedicated "resubmit" button, you can follow these steps to reverify your academic status:

- Go to the GitHub Education benefits page: [Link](https://education.github.com/pack)
- Click on your profile picture in the top right corner.
- Look for an option like "Reverify academic affiliation" or "Update academic information." It might be under "Settings" or "Account."

If you can't find the option:
Check the email you received after your appeal. It might contain a specific link for resubmitting your information.

If the email doesn't have a link:
Contact GitHub support directly. They can guide you through the process and troubleshoot any issues: [Visit](https://docs.github.com/)

**Reply by YvMiao1** [reply]
Thank you for your help, the first method probably requires a login to do it, and my account has been banned, I have used the second method and am waiting for a reply. Thank you.

**Reply by PingWang28** [reply]
I also meet the similar resumitting request,  I can find the resubmitting link , but when I click it , it always jump to  https://github.com/pricing , where I can't find a place to submit the information . Do you have some idea for this ?

**Reply by YvMiao1** [reply]
Sorry, I can't help you, I can't even find the link to resubmit

**Comment by mughal1999** [comment]
If you're having trouble finding where to resubmit your information, first check the website or app's navigation for any submission-related tabs or buttons. If you can't find it, try using the search function or contacting customer support for assistance. Also, review any email confirmations or previous submissions for guidance.

**Reply by YvMiao1** [reply]
Thanks, I'll give it a try
